### ISSUE TITLE
Generate client types from a GraphQL schema is at the wrong point in the document

### ISSUE THREAD

**Issue by qwikag** [aws-amplify/docs#5839]
**Describe the content issue**:
[Generate client types from a GraphQL schema](https://docs.amplify.aws/lib/graphqlapi/getting-started/q/platform/js/#generate-client-types-from-a-graphql-schema)

Is complete at the wrong point in the document further up it refers to
'.\graphql\mutations' etc andf this is not possible without the lower step.

for me the amplify push does not offer the step to include them.

**URL page where content issue is**:
[Generate client types from a GraphQL schema](https://docs.amplify.aws/lib/graphqlapi/getting-started/q/platform/js/#generate-client-types-from-a-graphql-schema)

**Comment by josefaidt** [comment]
Hey @qwikag :wave: thanks for raising this! After `amplify push` completes successfully it should update the corresponding GraphQL statements and types. Are you seeing an issue where this does not run after push? For local development, it may be beneficial to manually run `amplify codegen` to refresh your client-side code, however should also run during `amplify mock`

**Comment by qwikag** [comment]
Thanks @josefaidt,
I think my error might come from the fact that I installed a few months ago and maybe '>amplify push' did not offer that update to my mutations, or the install/inti process didn't???
because since I ran '>amplify codegen' it is now offering me that option, or at least I am recognising it.
but I cannot confirm this issue is resolved yet. I will be starting my 2nd project on next.js 13 and will be able to confirm, now that I have a deeper knowledge of what the commands and and when to use them, and what I am expecting.

Because the documentation is not complete, and lots of guides refer to per V13... as a noob it is very hard to know what to expect. It is a case of we don't know what we don't know. Add to that 13.4+ is so new, this makes for very challenging times.

I will update with my findings on my next install of next.

Your comments are helpful, and I will come back.
